src/partition/ReceivedPartition.cpp

```diff
@@ -279,12 +292,14 @@ void ReceivedPartition::filterByBoundingBox()
       PRECICE_ASSERT(utils::MasterSlave::getSize() > 1);
 
       for (int rankSlave = 1; rankSlave < utils::MasterSlave::getSize(); rankSlave++) {
-        com::CommunicateMesh(utils::MasterSlave::_communication).receiveBoundingBox(_bb, rankSlave);
+        mesh::Mesh::BoundingBox slaveBB(_bb.size(), std::make_pair(std::numeric_limits<double>::max(), std::numeric_limits<double>::lowest()));
+        com::CommunicateMesh(utils::MasterSlave::_communication).receiveBoundingBox(slaveBB, rankSlave);
 
-        PRECICE_DEBUG("From slave " << rankSlave << ", bounding mesh: " << _bb[0].first
-                                    << ", " << _bb[0].second << " and " << _bb[1].first << ", " << _bb[1].second);
+        PRECICE_DEBUG("From slave " << rankSlave << ", bounding mesh: "
+                                    << slaveBB[0].first << ", " << slaveBB[0].second << " and "
+                                    << slaveBB[1].first << ", " << slaveBB[1].second);
```

Comment:
In 3D, this debug statement feels a bit incomplete, but OK, it's just a debug statement.